### PR TITLE
Make the Enterprise login button more prominent

### PR DESCRIPTION
**Pull request by toolmantim**
Updates to the login styles as per [Google Doc](https://docs.google.com/document/d/1zdrQhLUTAx3mhV8qIUbMFXK5bgb-DGm6jpnKh96KRn0/edit#heading=h.fj8cdyv2vg9r).

| Before | After |
| - | - |
| Screenshot 2024-01-11 at 2 17 54 pm | Screenshot 2024-01-11 at 3 11 26 pm |

Notes:

* The secondary button color across VS Code themes often made it more prominent than the primary (the opposite of what we want), so the Enterprise login button was changed to be the same primary color as the others

## Test plan

- Viewed the login page
- Clicked the Enterprise login button
- Clicked the learn more link

### CODE REVIEW

**Review by abeatrix — APPROVED** [review]
Looks great ♥️

**Review by dominiccooney — APPROVED** [review]
Always a delight to see correct use of `&nbsp;`.

File: vscode/webviews/OnboardingExperiment.tsx [hunk 1]
```diff
@@ -119,13 +118,20 @@ export const LoginSimplified: React.FunctionComponent<React.PropsWithoutRef<Logi
...
                 </div>
-                <div className={styles.otherSignInOptions}>
-                    <h2>Using Sourcegraph Enterprise?</h2>
-                    <p>
-                        <button type="button" className={styles.linkButton} onClick={otherSignInClick}>
-                            Sign In to Your Enterprise Instance
-                        </button>
-                    </p>
+                <div className={styles.section}>
+                    <h1>Use Cody Enterprise</h1>
```
Comment (sqs) [inline vscode/webviews/OnboardingExperiment.tsx:122]: Perhaps removing "Use" from this and the other header would make it even clearer (just 1 less word).

**Review by sqs — APPROVED** [review]

File: vscode/webviews/OnboardingExperiment.tsx (hunk 1 above)
Comment (toolmantim) [reply vscode/webviews/OnboardingExperiment.tsx]: Yeah, I agree. It's also just awkward as a heading label. I've updated to remove "Use"

**Comment by rrhyne** [comment]
Thanks Tim!

**Pull request merged**